src/amo/components/Collection/index.js

```diff
@@ -0,0 +1,202 @@
+/* @flow */
+import React from 'react';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+
+import { withErrorHandler } from 'core/errorHandler';
+import log from 'core/logger';
+import translate from 'core/i18n/translate';
+import { parsePage } from 'core/utils';
+import { fetchCollection, fetchCollectionPage } from 'amo/reducers/collections';
+import type {
+  CollectionsState,
+  CollectionType,
+} from 'amo/reducers/collections';
+import AddonsCard from 'amo/components/AddonsCard';
+import Link from 'amo/components/Link';
```

Comment:
I find without sorting these imports just feel incredibly scattered and hard to sort through.

I think we should force imports to be ordered sooner than later...